code/modules/vending/clothesmate.dm

```diff
@@ -174,7 +174,10 @@
 		/obj/item/clothing/suit/basil_boys = 20,
 		/obj/item/clothing/under/costume/basil_boys = 20,
 		/obj/item/clothing/head/basil_boys = 20,
-		/obj/item/clothing/shoes/basil_boys = 20
+		/obj/item/clothing/shoes/basil_boys = 20,
+		/obj/item/clothing/suit/mothcoat = 3,
+		/obj/item/clothing/suit/mothcoat/winter = 3,
+		/obj/item/clothing/head/mothcap = 3
```

Comment:
```suggestion
		/obj/item/clothing/head/mothcap = 3,
```
Return the shopping cart, leave a trailing comma

Reply:
interesting, I didn't know you could do that